src/bootstrap/compile.rs

```diff
@@ -93,16 +93,16 @@ pub fn std_link(build: &Build,
     add_to_sysroot(&out_dir, &libdir);
 
     if target.contains("musl") && !target.contains("mips") {
-        copy_third_party_objects(build, target, &libdir);
+        copy_third_party_objects(build, &libdir);
     }
 }
 
 /// Copies the crt(1,i,n).o startup objects
 ///
 /// Only required for musl targets that statically link to libc
-fn copy_third_party_objects(build: &Build, target: &str, into: &Path) {
+fn copy_third_party_objects(build: &Build, into: &Path) {
```

Comment:
Can you change the name here to `copy_musl_third_party_object` as this only works for musl now?